build-tools/packages/build-cli/src/commands/generate/changelog.ts

```diff
@@ -22,7 +22,9 @@ async function replaceInFile(search: string, replace: string, path: string): Pro
 	await writeFile(path, newContent, "utf8");
 }
 
-export default class GenerateChangeLogCommand extends BaseCommand<typeof GenerateChangeLogCommand> {
+export default class GenerateChangeLogCommand extends BaseCommand<
+	typeof GenerateChangeLogCommand
+> {
```

Comment:
Just for the record, not a fan of this choice by Biome. Happens in several places. Not blocking on it though.

Reply:
FWIW, Prettier does the same thing when wrapping, Biome just seems to be being a tad more aggressive about line wrapping.